smoke-test-esm.mjs

```diff
@@ -0,0 +1,2 @@
+import TypeScriptLoader from "./dist/cjs/index.js";
+TypeScriptLoader()
```

Comment:
README.md currently shows that this works - however it does not as it uses a fake commonjs default export.